lib/util.js

```diff
@@ -128,6 +128,28 @@ exports.chooseFormat = (formats, options) => {
       format = formats[formats.length - 1];
       break;
 
+    case 'highestaudio':
+      var highest = formats[0];
+      for (var format of formats) {
```

Comment:
Use `let` for these two `var`s since these variables are redeclared in the next `case` block below